Project-Bootcamp/router.js

```diff
@@ -0,0 +1,37 @@
+module.exports = ( function () {
+    var React = require('react');
+    var Router = require('react-router');
+    var Route = Router.Route;
+    var DefaultRoute = Router.DefaultRoute;
+
+    var MainView = require ('./views/main-view');
+    var HeaderView = require ('./views/header-view');
+    var RealmView = require ('./views/realm-view');
+    var CharacterView = require ('./views/character-view');
+    var ChallengerView = require ('./views/challenger-view');
+    var PvpView = require ('./views/pvp-view');
+
+    var ModuleRouter = function () {
+
+        this.routes = (
+            <Route handler={MainView}>
```

Comment:
I think you should add a ``path`` here (``path="/"``). If you check [React Router's documentation](https://github.com/rackt/react-router/blob/master/docs/Introduction.md) it's always there in the main ``Route`` wrapper.